pymc3/variational/advi.py

```diff
@@ -303,7 +306,10 @@ def sample_vp(
     global_RVs = list(set(model.free_RVs) - set(rvs(local_RVs)))
 
     # Make dict for replacements of random variables
-    r = MRG_RandomStreams(seed=random_seed)
+    if random_seed is None:
+        r = MRG_RandomStreams()
```

Comment:
Oh, that's interesting -- I'm running [a test script](https://gist.github.com/ColCarroll/1f54f917cbe50ff9cd792e4af0f921ce), and it looks like the theano random number generator getting a seed doesn't effect the numpy random number generator, so you're right that we just need some way to get a different seed each time.

You can't actually [recover the numpy seed](http://stackoverflow.com/questions/32172054/how-can-i-retrieve-the-current-seed-of-numpys-random-number-generator), since there are too many states to encode in an int64.  I'll try (uniformly) randomly generating a seed using `numpy.random.randint`...